man/nvim.1

```diff
@@ -0,0 +1,417 @@
+.Dd May 7, 2015
+.Dt NVIM 1
+.Os Neovim
+.Sh NAME
+.Nm nvim
+.Nd edit text
+.Sh SYNOPSIS
+.Nm
+.Op Ar options
+.Op Ar file ...
+.Nm
+.Op Ar options
+.Fl
+.Nm
+.Op Ar options
+.Fl t Ar tag
+.Nm
+.Op Ar options
+.Fl q Ar errorfile
+.Sh DESCRIPTION
+.Nm
+is a text editor based on
+.Xr vim 1 .
+To enter commands in
+.Nm ,
+type a colon
+.Qq ( Cm \&: ) ,
+which is also used in this manual to denote commands.
+For more information, consult the on-line help system with the
+.Ic :help
+command.
+.Bl -tag -width Fl
+.It Ar file ...
+File(s) to edit.
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be associated with a
+terminal.
```

Comment:
I also think it's a hyper-correction to change this from the original wording, 

> Commands are read from stderr, which should be a tty.

The new form is totally meaningless to me. I don't know what "associated with a terminal" means, but I immediately understand "should be a tty". 

TTY is technical jargon with a specific meaning that is understood in context. No one in the context of a manpage mistakes it for physical teletypes.